crates/bevy_render/src/lib.rs

```diff
@@ -198,7 +198,7 @@ impl Plugin for RenderPlugin {
                     // flushing as "invalid" ensures that app world entities aren't added as "empty archetype" entities by default
                     // these entities cannot be accessed without spawning directly onto them
                     // this _only_ works as expected because clear_entities() is called at the end of every frame.
-                    render_app.world.entities_mut().flush_as_invalid();
+                    unsafe { render_app.world.entities_mut() }.flush_as_invalid();
```

Comment:
Is there a way to avoid this entirely?

Reply:
not sure, I dont particularly understand what the render app is actually trying to accomplish here :sweat_smile:

Reply:
the render apps "reserve" all the entities from the main app, but actually spawn something only on some of them. this marks all entities by default as invalid so that the one which won't be used aren't usable

Reply:
The "right" way to do this would be to expose a safe mirror of `flush_as_invalid()` on World. But honestly this feature is niche enough that im fine leaving this as-is for now.